### PR TITLE
chore: bump `flutter_hooks` to latest

### PR DESCRIPTION
**Pull request by saileshbro**

### CODE REVIEW

**Review by FilledStacks — DISMISSED** [review]

**Review by FilledStacks — APPROVED** [review]
thanks for fixing the CI. 

I'll push once the build works.

**Comment by FilledStacks** [comment]
Fixed the CI, let me try again. 

Analyze is failing, but that's because it has old code in there, which I don't want to update now lol

**Comment by saileshbro** [comment]
@FilledStacks looks like you forgot to push this to pub 😅

**Comment by FilledStacks** [comment]
Thought it would be automatic 🥸 Pushing now

**Comment by FilledStacks** [comment]
0.2.4 released 🚀
